core/src/main/scala/chisel3/properties/package.scala

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+/** The properties package includes functionality related to non-hardware data.
+  *
+  * This entire package is currently very experimental, so expect some rough
+  * edges and rapid API evolution.
+  */
+package object properties {}
```

Comment:
what is this for?

Reply:
I'm not sure if it's needed... I was just adding this boilerplate in case we want to define any functions in here later.